### PR TITLE
NN-3022 updates

### PR DESCRIPTION
**Pull request by JamieDraperUK**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -143,13 +143,13 @@
...
-    "husky": "^3.0.5",
+    "husky": "^4.3.0",
     "jest": "^24.8.0",
     "jest-styled-components": "^6.3.1",
-    "jest-useragent-mock": "^0.0.3",
+    "jest-useragent-mock": "^0.1.1",
     "jsdom": "^13.0.0",
     "lint-staged": "^8.2.1",
-    "mkdirp": "^0.5.1",
+    "mkdirp": "^1.0.4",
```
Comment (andrewrlee) [inline package.json:152]: I had problems with mkdirp as the interface changed and the documentation is a bit of a mess - I ended up removing this and using what we have in use-of-force: https://github.com/ministryofjustice/hmpps-book-video-link/pull/46

**Review by andrewrlee — DISMISSED** [review]
LGTM

File: package.json (hunk 1 above)
Comment (andrewrlee) [reply package.json:152]: (The problem was that it broke the build info scripts)

Comment (JamieDraperUK) [reply package.json:152]: Awesome, thanks Andy.

File: integration-tests/integration/alerts/editAlert.spec.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ context('A user can add an appointment', () => {
     cy.url().should('include', `prisoner/${offenderNo}/alerts`)
   })
 
-  it('Should show correct error messages', () => {
+  it.only('Should show correct error messages', () => {
```
Comment (petergphillips) [inline integration-tests/integration/alerts/editAlert.spec.js:40]: :scream:

Comment (petergphillips) [reply integration-tests/integration/alerts/editAlert.spec.js:40]: shows a fundamental issue with cypress and the inability to run a single test

File: src/utils.test.js [hunk 3]
```diff
@@ -141,15 +137,18 @@ describe('isAfterToday()', () => {
...
+    expect(isAfterToday(tomorrow)).toBe(true)
   })
 
   it('returns true if date is within the next week', () => {
     expect(isAfterToday('19/01/2019')).toBe(true)
   })
 })
 
-describe.only('isWithinNextTwoWorkingDays()', () => {
+describe('isWithinNextTwoWorkingDays()', () => {
```
Comment (JamieDraperUK) [inline src/utils.test.js:151]: :cry:

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**